packages/apollo-gateway/src/index.ts

```diff
@@ -239,37 +245,50 @@ export class ApolloGateway implements GraphQLService {
     };
   }
 
-  protected async updateComposition(options?: {
-    engine?: GraphQLServiceEngineConfig;
-  }) {
+  private async updateComposition() {
```

Comment:
It'd be nice to have return types on this.

Reply:
Resolved in eafff24